convert_state.py

```diff
@@ -1,60 +1,65 @@
+"""
+    This program converts between US state names, and abbreviations. 
+    For example, converting "MN" to "Minnesota" 
+    or converting "California" to "CA".
+"""
+import json
 
-
-
-import json  # import json for json processing
 def main():
+    """
+    Read data from JSON file to start main method
+    """
     with open('data.json','r') as file:
         data = file.read()
         data = json.loads(data)
```

Comment:
Use two different variables. Re-using data for a string read from a file, and then using it again for the dictionary is not good style. "data" is also a bad variable name - be more specific

Reply:
looks like Clara beat me to this @KelseyStiff

Reply:
@claraj  @svenJonis138 thanks for the feedback! I changed the variable names to be different and more descriptive.